build-system/tasks/markdown-toc/README.md

```diff
@@ -0,0 +1,77 @@
+# markdown-toc
```

Comment:
Methinks you should add an `OWNERS` file under `build-system/tasks/markdown-toc/` that lists you as an owner, and notifies you when someone changes the file.


```js
// For an explanation of the OWNERS rules and syntax, see:
// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example

{
  rules: [
    {
      owners: [
        {name: 'ampproject/wg-infra'},
        {name: 'alanorozco', notify: true},
      ],
    },
  ],
}

```